### ISSUE TITLE
Not able to get the results similar to the one mentioned in Readme

### ISSUE THREAD

**Issue by agarwalayush** [IAIK/cache_template_attacks#1]
Hi,
I am trying to generate the template-profiles of each key by running the `linux_low_frequency_example`, but the results are nowhere near to the ones mentioned in `Readme.md`. 
The `log.txt` thus generated doesn't have any address which has a sharp distinction in terms of `cache hits` for key `n`. While running the example, I tweaked the code to only look for `libgdk-3` files.
Can you please tell me what may have gone wrong, or what set of conditions did you use while experimenting?

Thanks

**Comment by dgruss** [comment]
maybe a different library version, then the arrays might have different offsets or even have been moved to different .so files... i would start with only a few keys first, that's faster and as soon as that works try other keys as well...

**Comment by agarwalayush** [comment]
Yes I always made sure that gedit was the target window.
First I tried with all the keys but only on a specific library which you have mentioned in the paper `/usr/lib/x86_64-linux-gnu/libgdk-3.so.0.1000.8`.
Then I tried with few specific keys `mnopq` and increased duration (400) but still not good results. 

And also, Currently I am using ubuntu's default window manager which has around 26 instances of libgdk-3 in the memory. Can that create a significant amount of noise?
